AEPServices/Sources/ui/fullscreen/FullscreenMessage.swift

```diff
@@ -91,14 +91,20 @@
         /// 2. check if the messageMonitor is allowing the message to be shown
         ///     a. if yes, show the message and exit the function
         ///     b. if no, call onShowFailure of the listener and exit the function
+        ///
+        /// - Parameter completion: called with `true` if message was shown
+        public func show(_ completion: @escaping ((Bool) -> Void)) {
```

Comment:
had to add a new method because this class conforms to the `Showable` protocol, requiring a `show()` method that takes no parameters